### PR TITLE
https://issues.redhat.com/browse/ACM-15011

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira: https://issues.redhat.com/browse/ACM-15011

See Dev PR where I took this content: https://github.com/stolostron/rhacm-docs/pull/7158/files

### CODE REVIEW

File: release_notes/known_issues_global.adoc [hunk 1]
```diff
@@ -32,6 +32,16 @@ If you import a managed hub cluster in the hosted mode and detach this managed h
 
 There is currently no workaround for this issue. 
 
+[#kafka-clusters]
+== Kafka clusters cannot deploy
+//2.12:15011
+
+When you re-create the {global-hub} custom resource definition (CRD), you might be unable to create or deploy the Kafka cluster instance. To workaround this issue, complete the following steps:
```
Comment (jc-berger) [inline release_notes/known_issues_global.adoc]: Dev reviewers:

Looking at the awesome PR that Chunlin opened, it seemed to me that the issue occurs when users are re-creating the global hub CRD. Correc tme if I am wrong.

File: release_notes/known_issues_global.adoc [hunk 2]
```diff
@@ -32,6 +32,16 @@ If you import a managed hub cluster in the hosted mode and detach this managed h
...
 There is currently no workaround for this issue. 
 
+[#kafka-clusters]
+== Kafka clusters cannot deploy
+//2.12:15011
+
+When you re-create the {global-hub} custom resource definition (CRD), you might be unable to create or deploy the Kafka cluster instance. To workaround this issue, complete the following steps: 
+
+. Manually restart the {global-hub} operator. 
+. Verify that you see the message, `wait for the Kafka CRD to be ready`.
```
Comment (jc-berger) [inline release_notes/known_issues_global.adoc]: Dev reviewers:

Is there a specific place in the global hub operator where users would go to see this message?

File: release_notes/known_issues_global.adoc [hunk 3]
```diff
@@ -32,6 +32,16 @@ If you import a managed hub cluster in the hosted mode and detach this managed h
 
 There is currently no workaround for this issue. 
 
+[#kafka-clusters]
+== Kafka clusters cannot deploy
```
Comment (clyang82) [inline release_notes/known_issues_global.adoc]: I would like to add `if recreating the global hub CR` to highlight it.

File: release_notes/known_issues_global.adoc (hunk 1 above)
Comment (clyang82) [reply release_notes/known_issues_global.adoc]: it is not CRD. it is a CR (Custom Resource)

File: release_notes/known_issues_global.adoc (hunk 2 above)
Comment (clyang82) [reply release_notes/known_issues_global.adoc]: sorry for misleading, `wait for the Kafka CRD to be ready` is in the global hub operator log. And it is not used to verify the success.  Actually, There are many that message in the log to indicate Kafka clusters cannot deploy

File: release_notes/known_issues_global.adoc [hunk 4]
```diff
@@ -32,6 +32,12 @@ If you import a managed hub cluster in the hosted mode and detach this managed h
 
 There is currently no workaround for this issue. 
 
+[#kafka-clusters-recreate-global-hub]
+== Kafka clusters cannot deploy if re-creating the {global-hub} custom resource 
+//2.12:15011
+
+If you re-create the {global-hub} custom resource (CR), you might be unable to create or deploy the Kafka cluster instance. To workaround this issue, you must manually restart the {global-hub} operator. In the {global-hub} operator log, you can see a message, informing you that the Kafka custom resource definition (CRD) is ready. For example, you might see a message such as, `wait for the Kafka CRD to be ready`.
```
Comment (dockerymick) [inline release_notes/known_issues_global.adoc]:
```suggestion
If you re-create the {global-hub} custom resource, you might be unable to create or deploy the Kafka cluster instance. To workaround this issue, you must manually restart the {global-hub} operator. In the {global-hub} operator log, you can see a message, informing you that the Kafka custom resource definition is ready. For example, you might see a message such as, `wait for the Kafka CRD to be ready`. 
```

**Review by dockerymick — APPROVED** [review]
left a comment. I believe the acronyms in parentheses is not needed. Otherwise, this looks good to me! I did notice that throughout our product doc, we are using "recreate", which is incorrect based on the Style guide: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#r

We can create a refresh doc issue to update those later. The way you have it hyphenated in this PR is correct. Great work!

File: release_notes/known_issues_global.adoc (hunk 4 above)
Comment (clyang82) [reply release_notes/known_issues_global.adoc]: If you re-create the {global-hub} custom resource, you might be unable to create or deploy the Kafka cluster instance. In the {global-hub} operator log, you may see a recurring message: `waiting for the Kafka CRD to be ready`. To work around this issue, you must manually restart the {global-hub} operator.

**Comment by dockerymick** [comment]
This looks good to me!

**Pull request merged**